src/resources/startup.py

```diff
@@ -305,7 +305,7 @@ def Execute(self, request, context):
                 output_shape = output_np_array.shape
 
                 # We need to serialize TYPE_STRING
-                if output_np_array.dtype.type is np.object or output_np_array.dtype.type is np.bytes_:
+                if output_np_array.dtype.type is np.object_ or output_np_array.dtype.type is np.bytes_:
```

Comment:
As you suggested, please change this to `output_np_array.dtype == np.object or output_np_array.dtype.type is np.bytes_`.